packages/teams-js/test/promiseTester.ts

```diff
@@ -0,0 +1,27 @@
+export enum PromiseState {
+  Pending = 'pending',
+  Resolved = 'resolved',
+  Rejected = 'rejected',
+}
+
+export async function getPromiseState<T>(promiseInQuestion: Promise<T>): Promise<PromiseState> {
```

Comment:
Generally speaking, "utils" files are a dumpster for all kinds of random unrelated code and end up growing in an unmaintainable fashion. I put this into it's own file so that the purpose of the code in this file was clear and it could remain single-purpose. If we have other helpful utilities related to promises or async validation, they could also go into this file since they would be united under a single purpose.